WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -408,7 +410,10 @@ class AztecPostViewController: UIViewController, PostEditor {
     fileprivate lazy var verificationPromptHelper: AztecVerificationPromptHelper? = {
         return AztecVerificationPromptHelper(account: self.post.blog.account)
     }()
-
+    
+    // Autosaving posts 
+    fileprivate var shouldAutoSave: Bool = false
```

Comment:
I was going to say there should be at the top of the file but then I realised this file has over 400 lines of properties. Holy cow!